_layouts/styleguide.html

```diff
@@ -16,13 +16,15 @@
     <header>
       {% if page.category != null %}
         {% unless page.path contains 'overview' %}
-      <p class="site-subheading">{{ page.category }}</p>
+          <p class="site-subheading">{{ page.category }}</p>
         {% endunless %}
       {% endif %}
       <h1 id="{{ page.title | slugify }}">{{ page.title }}</h1>
-      <span class="components-library-link">
-        <a href="{{ page.component_url }}">View in Component Library</a>
-      </span>
+      {% if page.category == 'UI components' and page.fractal_component_url %}
```

Comment:
Oh interesting, it's not enough to just test if `page.fractal_component_url` is truthy here? What's the situation where it isn't enough? I recall you mentioning on Slack that you didn't realize that this layout was used on non-components pages, but I would've thought that if it was used elsewhere, `page.fractal_component_url` would be falsy...